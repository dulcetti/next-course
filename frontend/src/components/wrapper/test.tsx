import { render, screen } from '@testing-library/react';
import Wrapper from '.';

describe('Container Component', () => {
  it('should be render the Heading', () => {
    const { container } = render(<Wrapper />);

    expect(screen.getByRole('heading', { name: /papo de bar/i })).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
