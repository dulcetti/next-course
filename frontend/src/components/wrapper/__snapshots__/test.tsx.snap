// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component should be render the Heading 1`] = `
<main
  class=""
>
  <h1
    class=""
  >
    Papo de Bar
  </h1>
</main>
`;
